include/Reader/readerir.h

```diff
@@ -558,6 +560,12 @@ class GenIR : public ReaderBase {
 
   EHRegion *rgnAllocateRegion() override;
   EHRegionList *rgnAllocateRegionList() override;
+  void setDebugLocation(uint32_t CurrOffset, uint32_t IsCall) override;
```

Comment:
Or maybe a enum with `IsCall` and `IsStackEmpty` the two possible values? Then the uses are self-documenting.